packages/imperative/__tests__/config/cmd/convert-profiles/convert-profiles.handler.test.ts

```diff
@@ -189,6 +191,188 @@ describe("Configuration Convert Profiles command handler", () => {
+        });
+        const updateSchemaSpy = jest.spyOn(ConfigSchema, "updateSchema").mockReturnValueOnce(undefined);
+        jest.spyOn(fs, "renameSync").mockReturnValueOnce();
+        jest.spyOn(keytar, "findCredentials").mockResolvedValue([
+            {account: "testAcct", password: "testPassword"}
+        ]);
+        jest.spyOn(keytar, "deletePassword").mockResolvedValue(true);
+        const rimrafSpy = jest.spyOn(rimraf, "sync").mockImplementation(() => {
+            return true;
+        });
+
+        const handler = new ConvertProfilesHandler();
+        jest.spyOn(handler as any, "getObsoletePlugins").mockReturnValueOnce([]);
+        jest.spyOn(handler as any, "getOldProfileCount").mockReturnValueOnce(3);
+        const findOldSecurePropsSpy = jest.spyOn(handler as any, "findOldSecureProps");
+        const deleteOldSecurePropsSpy = jest.spyOn(handler as any, "deleteOldSecureProps");
+
+        const params = getIHandlerParametersObject();
+        params.arguments.force = true;
+        params.arguments.delete = true;
+        params.arguments.forSure = true;
+
+        await handler.process(params);
+        expect(stdout).toContain("Detected 3 old profile(s)");
+        expect(stdout).toContain("Converted fruit profiles: apple, coconut, banana");
+        expect(stdout).toContain("Deleting the profiles directory");
+        expect(stdout).toContain("Deleting secure value for \"@brightside/core/testAcct\"");
+        expect(stdout).toContain("Deleting secure value for \"@zowe/cli/testAcct\"");
+        expect(stdout).toContain("Deleting secure value for \"Zowe-Plugin/testAcct\"");
+        expect(stdout).toContain("Deleting secure value for \"Broadcom-Plugin/testAcct\"");
+        expect(stdout).toContain("Deleting secure value for \"Zowe/testAcct\"");
+        expect(updateSchemaSpy).toHaveBeenCalled();
+        expect(mockImperativeConfig.config.save).toHaveBeenCalled();
+        expect(rimrafSpy).toHaveBeenCalledTimes(1);
+        expect(findOldSecurePropsSpy).toHaveBeenCalledTimes(5);
+        expect(deleteOldSecurePropsSpy).toHaveBeenCalledTimes(5);
+    });
+
+    it("should remove existing profiles and delete secure properties except secure_config_props", async () => {
+        const metaError = new Error("invalid meta file");
+        jest.spyOn(ConfigBuilder, "convert").mockResolvedValueOnce({
+            config: Config.empty(),
+            profilesConverted: {
+                fruit: ["apple", "coconut", "banana"]
+            },
+            profilesFailed: []
+        });
+        const updateSchemaSpy = jest.spyOn(ConfigSchema, "updateSchema").mockReturnValueOnce(undefined);
+        jest.spyOn(fs, "renameSync").mockReturnValueOnce();
+        jest.spyOn(keytar, "findCredentials").mockResolvedValue([
+            {account: "testAcct", password: "testPassword"},
+            {account: "secure_config_props", password: "testPassword"},
+            {account: "secure_config_props_1", password: "testPassword"}
```

Comment:
The DefaultCredentialManager uses a hyphen before the number at the end like `secure_config_props-1`. Should we update this account string to match?

Reply:
We can, it will still match, but better to be more realistic in the tests.